GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -1625,10 +1625,12 @@ private void InitializeComponent()
             this.Controls.Add(this.toolPanel);
             this.Controls.Add(this.menuStrip1);
             this.Controls.Add(this.statusStrip);
+            this.KeyPreview = true;
```

Comment:
This may introduce some overheads as noted in https://stackoverflow.com/a/2386980/2338036